src/Partition.hxx

```diff
@@ -88,6 +88,12 @@ struct Partition final : QueueListener, PlayerListener, MixerListener {
 		return playlist.AppendURI(pc, loader, uri_utf8);
 	}
 
+	unsigned AppendURI(const SongLoader &loader,
```

Comment:
This whole commit is obsolete, because MPD has a better implementation for this already.

Reply:
Just for caryaudio.